sickbeard/webserve.py

```diff
@@ -443,15 +443,6 @@ def setHistoryLayout(self, layout):
 
         return self.redirect("/history/")
 
-    def setHistoryLimit(self, history_limit):
-
-        if not int(history_limit):
-            history_limit = 100
-
-        sickbeard.HISTORY_LIMIT = history_limit
```

Comment:
It set it here
`sickbeard.HISTORY_LIMIT = history_limit`
this value is then reused for any page loads after that, and saved to ini when SR shuts down or anything other method saves config before then.